hyperspy/learn/mva.py

```diff
@@ -477,50 +479,15 @@ def decomposition(
                         data_, rank=output_dimension, store_error=True, **kwargs,
                     )
                     to_return = (X, E)
-                else:
-                    W, H = ornmf(data_, rank=output_dimension, **kwargs)
-
-                loadings = W
-                factors = H.T
-
-            elif is_sklearn_like:
```

Comment:
Something wrong with the merge @jat255 if this has been deleted, this looks wrong

Reply:
Oops, looks like you're right. This was a complicated merge, so I'm not surprised something went south. I'm working on getting the tests running again, which would have revealed this issue.